services/brig/test/integration/API/User/Client.hs

```diff
@@ -121,7 +128,7 @@ tests _cl _at opts p db b c g =
       test p "get/head nonce/clients" $ testNewNonce b,
       testGroup
         "post /clients/:cid/access-token"
-        [ test p "invalid values" $ testCreateAccessTokenInvalidValues b,
```

Comment:
why is this test removed?

Reply:
It is really the most trivial test and doesn't prove much value.